src/lib/csvParser.js

```diff
@@ -0,0 +1,116 @@
+import parseCsv from "csv-parse/lib/sync";
+import camelize from "camelize";
+
+export const parseData = csv => {
+  const jsResults = camelize(
+    parseCsv(csv, {
+      columns: true,
+      skip_empty_lines: true
+    })
+  );
+
+  /*
+    { precinctData: { [precinctId]: dataForPrecinct }}
+  */
+  const precinctResults = jsResults.reduce((acc, current) => {
+    const precinctData = rowToData(current);
+    return { ...acc, [precinctData.key]: precinctData };
+  }, {});
+
+  return {
+    precinctResults
+  };
+};
+
+// Replace later if needed- looking for a standard way to
+const precinctIdentifier = rowObject => {
+  const { precinctFull } = rowObject;
+  return `${precinctFull}`;
+};
+
+const rowToData = rowObject => {
+  const key = precinctIdentifier(rowObject);
+  const {
+    // Issues - keep up to date w/ csv model
+    viableLoss,
+    nonviableNoRealign,
+    alphaShift,
+    hasAlphaShift,
+    moreFinalVotes,
+    fewerFinalVotes,
+    delCountsDiff,
+    extraDelGiven,
+    // End Issues
+
+    ...cleanData
+  } = rowObject;
```

Comment:
This section is intended to pull out the things we don't want in the top level of a precinct's data (the errors, which will be massaged into `issues` below).